__tests__/unittest/extendOrdWithCustom.test.js

```diff
@@ -45,9 +44,13 @@ describe('extendOrdWithCustom', () => {
 
         it('should ignore and log warn if found ord top-level primitive property in customOrdFile', () => {
             const ordContent = {};
+            const warningSpy = jest.spyOn(console, 'warn');
             prepareTestEnvironment({ namespace: "sap.sample" }, appConfig, 'testCustomORDContentFileThrowErrors.json');
-            const result = extendCustomORDContentIfExists(appConfig, ordContent, logger);
-            expect(logger.warn).toHaveBeenCalledTimes(3);
+            const result = extendCustomORDContentIfExists(appConfig, ordContent);
+
+            expect(warningSpy).toHaveBeenCalledTimes(3);
+            expect(warningSpy).toHaveBeenCalledWith('Mocked warning');
+
```

Comment:
@zongqichen : This way we can also test if the `cds.log.warn` is called in function `compareAndHandleCustomORDContentWithExistingContent`. 
Benefit of this way of testing is that we do not need to propagate the `logger` through 3 functions. So, this way we don't need parameter `logger` is the following functions:
```
extendCustomORDContentIfExists
compareAndHandleCustomORDContentWithExistingContent
```